File: java/fibonacci.java
import java.util.*;
public class fibonacci {
    public static void main(String[] arg){
        int num , a=0, b=0, c=1;
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter any number : ");
        num = sc.nextInt();
        System.out.print("Fibonacci series till " +num+ " is ");

        for(int i = 1; i <= num; i++){
            a = b;
            b = c;
            c = a + b;
            System.out.print(a + " ");



        }



sc.close();
    }
    
}

File: python/fibonacci.py
num = int(input("Enter the number : "))
a , b= 0, 1
count = 0
if num <= 0:
    print("enter any positive number ")
elif num == 1:
    print("fabonacci series is 1 ", num)
    print(a)
else:
    print("fabonacci series of ", num, " is : ")
    while count < num:
        print(a)
        c = a + b
        a = b
        b = c
        count += 1
    